ticker/src/main/java/com/robinhood/ticker/TickerColumn.java

```diff
@@ -115,7 +115,7 @@ float getMinimumRequiredWidth() {
      * A helper method for populating {@link #startIndex} and {@link #endIndex} given the
      * current and target characters for the animation.
      */
-    private void setCharacterIndices() {
+    private void setCharacterIndices(boolean wraparound) {
```

Comment:
i've just taken a quick glance at this, will do a full review tomorrow. just wanna ask - does the wraparound always force an upward animation? if i'm going from `9` to `8`, i'm assuming we still animate downwards?

Reply:
yep, it will always wrap-around and animate upwards.